yt_dlp/extractor/polskieradio.py

```diff
@@ -48,28 +50,11 @@ def _extract_webpage_player_entries(self, webpage, playlist_id, base_data):
             yield entry
 
 
-class PolskieRadioIE(PolskieRadioBaseExtractor):
-    _VALID_URL = r'https?://(?:www\.)?polskieradio(?:24)?\.pl/\d+/\d+/Artykul/(?P<id>[0-9]+)'
-    _TESTS = [{  # Old-style single broadcast.
-        'url': 'http://www.polskieradio.pl/7/5102/Artykul/1587943,Prof-Andrzej-Nowak-o-historii-nie-da-sie-myslec-beznamietnie',
-        'info_dict': {
-            'id': '1587943',
-            'title': 'Prof. Andrzej Nowak: o historii nie da się myśleć beznamiętnie',
-            'description': 'md5:12f954edbf3120c5e7075e17bf9fc5c5',
-        },
-        'playlist': [{
-            'md5': '2984ee6ce9046d91fc233bc1a864a09a',
-            'info_dict': {
-                'id': '1540576',
-                'ext': 'mp3',
-                'title': 'md5:d4623290d4ac983bf924061c75c23a0d',
-                'timestamp': 1456594200,
-                'upload_date': '20160227',
-                'duration': 2364,
-                'thumbnail': r're:^https?://static\.prsa\.pl/images/.*\.jpg$'
-            },
-        }],
-    }, {  # New-style single broadcast.
```

Comment:
Why are the tests removed?

Reply:
moved from the legacy page extractor to the new frontend extractor below, as the site it was on got redesigned.
https://github.com/yt-dlp/yt-dlp/pull/5416/files/9190f6e541ec280ffa77b1dcde21a5462731275a#diff-7015cf2e8c40def45004b56bfcf2149d62f465abea11c89ddf817dab3c2f08b1R138